styleguide.md

```diff
@@ -0,0 +1,180 @@
+
+When referencing a named function of operator, hyphenation is not used, and it is common to not combine with any describing word:
+
+Appearance | LaTeX source
+--- | ---
+the `smooth` operator | `the \lstinline!smooth! operator`
+where `pre` is not explicitly used | `where \lstinline!pre! is not explicitly used`
+
+### Other things, special cases
+
+Incomplete list of various terminology with special formatting rules:
+
+Appearance | LaTeX source | Comment
+--- | --- | ---
+start value | `start value` | Value of the `start`-attribute (there could be exceptions!)
+reduction expression | `reduction expression` |
+base class | `base class` | Similarly: derived class
+base-clock | `base-clock` | Similarly: sub-cock
+
+## Definitions
+
+New terminology is either introduced with a `definition` environment, or as part of the running text.
+When part of the running text, the introduced terminology should be marked with `\firstuse` at the point of the definition.
+(The use of `\firstuse` helps us both produce consistent formatting and keep track of things that should appear in the document index.)
+
+If the new terminology is used before being introduced, the first use should be marked with `\willintroduce` to alert the reader that this is not a term that is expected to be known yet by a first-time reader.
+
+## Miscellaneous
+
+### Use of `\emph` and italics
+
+To put emphasis on a word or small piece of text, use `\emph`.
+
+Italics is used when new terminology is introduced in the running text instead of the bulkier `definition` environment, see `\firstuse` and `\willintroduce`.
+
+Non-semantical font switching commands for producing italics (`\textit`, `\textsl`, `\itshape`) should pretty much never be used.
+
+Note that the document is full of text set in italics, since this is used for both non-normative text and examples, through the `nonnormative` and `example` environments.
+
+### Use of boldface
+
+Non-semantical font switching commands for producing boldface (`\textbf`, `\bfseries`) may only be used for styling as part of higher level semantic constructs such as the _amsthm.sty_ `\newtheoremstyle` definitions.
+For purposes of marking emphasis, see use of `\emph` instead.
+
+### Ordinals
+
+Ordinals are written with _th_ in normal font, possibly combined with a math styled expression for the number:
+```
+Fixed ordinals: 1st, 2nd, 3rd.
+Symbolic ordinals: $n$th, $(n+1)$th.
+```
+
+## Code listings
+
+### Modelica listings
+
+Modelica listings are written with indentation in steps of 2 spaces.
+
+A space is typically added on each side of a binary operator, and after comma.
+
+Code snippets may start with an indented line as long as there's some line in the listing with zero indentation, like this:
+```
+  Real x(start = 1.0, fixed = true);
+equation
+  der(x) = -x;
+```
+
+Each line should fit within the width of the page, using hard line breaks and manual indentation of continued lines to meet this requirement.
```

Comment:
"using hard line breaks...": It should either be "use" or should be followed by "is recommended" or "is required".

Reply:
I agree, and I would say prefer 'use'

Reply:
OK, I made it two sentences.  OK?